gnd/src/test/java/com/google/android/gnd/persistence/local/DateTimeResponseTest.java

```diff
@@ -16,39 +16,36 @@
 
 package com.google.android.gnd.persistence.local;
 
+import com.google.android.gnd.model.observation.DateResponse;
+import com.google.android.gnd.model.observation.TimeResponse;
 import com.google.common.truth.Truth;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
+import java.time.Instant;
 import java.time.LocalDate;
-import java.time.LocalTime;
-import java.time.format.DateTimeFormatter;
+import java.time.Month;
+import java.time.ZoneId;
 import java.util.Date;
-import java.util.Locale;
 import org.junit.Test;
 
 public class DateTimeResponseTest {
 
-  public final DateFormat timeFormat = new SimpleDateFormat("HH:mm", Locale.getDefault());
-  public final DateFormat dateFormat =
-      new SimpleDateFormat("yyyy-MM-dd", Locale.getDefault());
+  // Sat, 23 Oct 2021 07:30:45 (Local Time)
+  private static final Instant EXPECTED_INSTANT =
+      LocalDate.of(2021, Month.OCTOBER, 23)
+          .atTime(7, 30, 45)
+          .atZone(ZoneId.systemDefault())
+          .toInstant();
+
+  private Date getExpectedDate() {
+    return Date.from(EXPECTED_INSTANT);
+  }
 
   @Test
-  public void testTimeFormat() {
-    Date originalDate = new Date();
-    String timeString = timeFormat.format(originalDate);
-    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm", Locale.getDefault());
-    LocalTime lt = LocalTime.parse(timeString, formatter);
-    String formattedTime = lt.format(formatter);
-    Truth.assertThat(timeString).isEqualTo(formattedTime);
+  public void testTimeResponse_textDetails() {
+    Truth.assertThat(new TimeResponse(getExpectedDate()).getDetailsText()).isEqualTo("07:30");
```

Comment:
For self-describing methods like "assertThat", it's better for readability to just static import the method instead.

Reply:
It's probably not important since this class is small, but I would prefer to have the time showing up somewhere in this test method to show the readers on why we are expecting "07:30" text.

Reply:
Done